packages/mock-app/src/__tests__/JsonBlock.test.tsx

```diff
@@ -34,29 +33,14 @@ describe('Json block content', () => {
     },
   };
 
+  const renderComponent = (verifiableCredential: any) => {
+    return render(<JsonBlock credential={credential} />);
+  };
+
   it('should render json block content component', () => {
     // Render the JsonBlock component with the mock credential
-    render(<JsonBlock credential={credential} />);
+    renderComponent(credential);
     // Expecting the text 'VerifiableCredential' to be present in the rendered component
     expect(screen.getByText(/VerifiableCredential/i)).not.toBeNull();
   });
```

Comment:
Could you write additional test cases for the JsonBlock component to verify how it renders the decodedEnvelopedVC?

Reply:
Actually, there is no difference between decodedEnvelopedVC and credential VC when they are passed as props from here. It's a JSON with data like this but delete proof field with decodedEnvelopedVC:
<img width="791" alt="image" src="https://github.com/user-attachments/assets/70c4f011-e155-43a3-9c66-bfcb1a757253">